tests/lib/cli-engine/lint-result-cache.js

```diff
@@ -163,16 +163,21 @@ describe("LintResultCache", () => {
             it("contains node version during hashing", () => {
                 const version = "node-=-version";
 
-                sandbox.stub(process, "version").value(version);
-                const NewLintResultCache = proxyquire("../../../lib/cli-engine/lint-result-cache.js", {
-                    "./hash": hashStub
-                });
-                const newLintResultCache = new NewLintResultCache(cacheFileLocation, "metadata");
+                const versionStub = sandbox.stub(process, "version").value(version);
```

Comment:
This wasn't correctly restored, causing `process.version` to always be the mocked value for sub-sequence tests.

Reply:
Oh my gosh, thank you so much for figuring this one out, I was banging my head on the wall for quite some time trying to figure this one out.